src/sass/app/sell_popup.scss

```diff
@@ -1,3 +1,4 @@
+/* styline-disable */
```

Comment:
what is styline? why do we need to disable it?

Reply:
We don't, it's remnant that needs to be removed. 👍